prov/gni/src/gnix_auth_key.c

```diff
@@ -386,24 +388,20 @@ int _gnix_auth_key_insert(
 		return -FI_EINVAL;
 	}
 
-	if (auth_key_size == GNIX_PROV_DEFAULT_AUTH_KEYLEN)
-		key = 0;
-	else {
-		if (!auth_key) {
-			GNIX_INFO(FI_LOG_FABRIC, "auth key is null\n");
-			return -FI_EINVAL;
-		}
+	if (!auth_key) {
```

Comment:
I think this would break the logic that was here originally. The default auth key references the protection key assigned to the job by the WLM.

Reply:
Nevermind, I see how it was changed to handle that condition.